src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml

```diff
@@ -81,7 +81,7 @@ tests:
                 value: null
 
     - label:
-          "TH reads NumberOfTotalUsersSupported attribute and saves for future
+          "TH reads NumberOfTotalUsers Supported attribute and saves for future
```

Comment:
Why this change?  The attribute is called NumberOfTotalUsersSupported.

Reply:
my mistake i will fix this